lib/portage/const.py

```diff
@@ -181,6 +181,7 @@
         "digest",
         "distcc",
         "distlocks",
+        "dont-export-pms-vars",
```

Comment:
Could we make this `export-pms-vars` and enable it by default?

Reply:
Unfortunately, I don't think so, as there is no way to disable a feature. Or does `FEATURES="-export-pms-vars"` work (havn't checked)?

The FEATURE=dont-export-pms-vars is there to test the behavior with current EAPIs. That is, it is opt-in, instead of opt-out. In theory, we could also remove the dont-export-pms-vars FEATURE once we are reasonably convinced that it does not break anything.

Reply:
Yes, FEATURES="-x" works.

Reply:
Ok, not sure if I missed it in make.conf's man page, but if it isn't there, then it should potentially be documented.

So what would
```python
 if "dont-export-pms-vars" in mysettings.features or not eapi_exports_pms_vars(eapi):
```

become with FEATURES="-export-pms-vars"? Simply

```python
 if "-export-pms-vars" in mysettings.features or not eapi_exports_pms_vars(eapi):
```
?

Reply:
It would be `"export-pms-vars" not in mysettings.features`.